frontend/_base.js

```diff
@@ -35,34 +35,50 @@ module.exports = {
     // Enforce component methods order
     'react/sort-comp': [2, {
       'order': [
-        'displayName',
-        'propTypes',
-        'contextTypes',
-        'childContextTypes',
-        'mixins',
-        'statics',
-        'defaultProps',
-        'state',
-        'constructor',
-        'getDefaultProps',
-        'getInitialState',
-        'getChildContext',
-        'componentWillMount',
-        'componentDidMount',
-        'componentWillReceiveProps',
-        'shouldComponentUpdate',
-        'componentWillUpdate',
-        'componentDidUpdate',
-        'componentWillUnmount',
+        'static-methods',
+        'lifecycle',
         '/^(on|handle).+$/',
         '/^get.+$/',
         'everything-else',
-        '/^render.+$/',
-        'render',
+        'rendering',
       ],
+      'groups': {
+        'lifecycle': [
+          'displayName',
+          'propTypes',
+          'contextTypes',
+          'childContextTypes',
+          'mixins',
+          'statics',
+          'defaultProps',
+          'constructor',
+          'getDefaultProps',
+          'getInitialState',
+          'state',
+          'getChildContext',
+          'componentWillMount',
+          'componentDidMount',
+          'componentWillReceiveProps',
+          'shouldComponentUpdate',
+          'componentWillUpdate',
+          'componentDidUpdate',
+          'componentWillUnmount',
+        ],
+        'rendering': [
+          '/^render.+$/',
+          'render',
+        ],
+      },
     }],
 
     /**
+     * Compatibility from 1.0.0 -> 2.0.0 migration, mostly new rules
+     * from Airbnb config that conflict with our existing code base.
+     */
+    // Prevent function creation on each render
+    'react/jsx-no-bind': 1, // intended: 2
```

Comment:
Talked with @levithomason about this, we _should_ be at a 2 but until we fix occurrences we're just going to keep it as a warning.